source/adapters/hip/command_buffer.cpp

```diff
@@ -884,7 +884,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferEnqueueExp(
     std::unique_ptr<ur_event_handle_t_> RetImplEvent{nullptr};
     ScopedContext Active(hQueue->getDevice());
     uint32_t StreamToken;
-    ur_stream_quard Guard;
+    ur_stream_guard Guard;
```

Comment:
Do you know why this code worked/compiled before this change?

Reply:
The type was declared and used with this typo. See I have changed the declaration here https://github.com/oneapi-src/unified-runtime/pull/1645/files#diff-8a960acc065ee19759fae311b3e8f329529a4a0863d22b5bdaa5345d9d6e1af9R17